en/console-commands/commands.rst

```diff
@@ -240,6 +240,35 @@ You may need to call other commands from your command. You can use
     parent command's ``ConsoleIo`` instance as the optional 3rd argument to
     avoid a potential "open files" limit that could occur in some environments.
 
+Setting Command Description
+===========================
+
+You may want to set a command description via::
+
+    class UserCommand extends Command
+    {
+        public static function getDescription(): string
+        {
+            return 'My custom description';
+        }
+    }
+
+This will show your description in the Cake CLI::
```

Comment:
This needs to use a shell code block

Reply:
```rst
... code-block:: console 

    bin/cake

```
is what you want.

Reply:
I first thought you meant adding a separte doc for CakePHP Shells, not the RST "shell" code styling for that block 😅 thanks markstory 😄

Reply:
it's supposed to be two . not three. `.. code-block::`.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block